### PR TITLE
Add Vue.js recipe

### PR DESCRIPTION
**Pull request by auchenberg**
Add recipe for Vue.js based on vue-cli. Fixes https://github.com/Microsoft/vscode-recipes/issues/55

Please have a look @octref

### CODE REVIEW

File: vuejs-cli/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Vue.js debugging in VS Code with Chrome and vue-cli
```
Comment (octref) [inline vuejs-cli/README.md]: I'd say more like `in VS Code and Chrome`. Don't feel vue-cli needs to get mentioned in the title.

Comment (octref) [reply vuejs-cli/README.md]: Also, maybe show some synchronized stepping or show that you can also inspect / F10 in Chrome (which is cool).

**Comment by sdras** [comment]
Short and sweet. I love it. Would you mind if I cross-posted something similar (though it would have to be more fleshed-out and follow our template, complete with alternate patterns) for the Vue cookbook in the official Vue docs?

**Comment by auchenberg** [comment]
@sdras Sure, the more docs the better as long as they are all linked up. @octref and I have been thinking about adding the debug config to the popular https://github.com/vuejs/vetur VS Code extension, so we can push it out to existing users.

**Comment by octref** [comment]
Official doc sounds great too, since I think this debugging complements (instead of replaces) Vue devtools debugging.
It would be great to have a doc that talks about multiple ways of debugging Vue app and when should each one be used.

**Comment by sdras** [comment]
Alright! All of these are great ideas- Everyone loves vetur, so I'm sure that will get some good eyes on it. I'll work away on the cookbook example and tag you in the PR when it's complete. I'm slammed right now so it might be a month or so. Thanks!

**Pull request merged**